rules-tests/TypeDeclaration/Rector/ClassMethod/ReturnTypeFromStrictNewArrayRector/Source/OrderRepository.php

```diff
@@ -0,0 +1,12 @@
+<?php
+declare(strict_types=1);
+
+namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\ReturnTypeFromStrictNewArrayRector\Source;
+
+final class OrderRepository
+{
+    public function fetchAllForBuyer(): array
+    {
+        return [];
+    }
+}
```

Comment:
Could you add test fixture with docblock array return type, that is skipped? Just to be sure, thanks

Reply:
This is source file under `Source` directory, not fixture file, so it doesn't add docblock.

Reply:
I see, I will add it.

Reply:
@TomasVotruba here the test fixture for skip docblock return:

- https://github.com/rectorphp/rector-src/pull/4933

Reply:
Thank you 🙏